integrators/methods.hpp

```diff
@@ -701,25 +692,13 @@ struct McLachlanAtela1992Order5Optimal : SymplecticRungeKuttaNyström {
                                                    -0.0589796254980311632}}};
 };
 
-// This integrator goes by many names, see Hairer, Lubich, and Wanner (2003),
-// Geometric numerical integration illustrated by the Störmer–Verlet method,
-// http://www.math.kit.edu/ianm3/lehre/geonumint2009s/media/gni_by_stoermer-verlet.pdf,
-// section 1.3 (Historical remarks).
+// This integrator goes by many names, see [HLW03], section 1.3 (Historical
+// remarks).
 // Notably, it appears in Philosophiae Naturalis Principia Mathematica,
```

Comment:
Should we have [New87]?

Reply:
Done, also with a reference in `ephemeris_body.hpp`.  Let's hope that we don't have to reference a paper by Newhall from 1987...